sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -138,6 +147,23 @@ struct {
 Supported since version 3 of this specification.[^1]
 </td>
 </tr><tr>
+<td>
+
+``` C++
+struct {
+  std::variant<ze_command_queue_handle_t,
+               ze_command_list_handle_t>
+    NativeHandle;
+  device Device;
+  ext::oneapi::level_zero::ownership Ownership{
+      ext::oneapi::level_zero::ownership::transfer};
+  property_list Properties{};
+}
+```
+
+Supported since version 4 of this specification.[^1]
+</td>
+</tr><tr>

```

Comment:
It seems like we should have some description about the two options for `NativeHandle` when calling `make_queue`.  Let's add a paragraph after the table that says something like:

> A SYCL queue can be created with `make_queue()` by passing either a Level Zero `ze_command_queue_handle_t` or a Level Zero `ze_command_list_handle_t`.  Queues created from a Level Zero immediate command list (`ze_command_list_handle_t`) generally perform better than queues created from a standard Level Zero `ze_command_queue_handle_t`.  See the Level Zero documentation of these native handles for more details.

If we have more details to say, we can include that too.  I'm not sure if we have any more details, though.  Do we?

We should also describe the new `Properties` member:

> The `make_queue()` function also accepts a `Properties` member variable.  This can contain any of the SYCL properties that are accepted by the SYCL `queue` constructor.

Reply:
Oh, I just remembered that we have section "Construct a SYCL object from a Level-Zero handle", which describes each of the "make_OBJECT" functions.  We should move these paragraph describing `make_queue` there.